storybookjs/storybook
# UI: Fix button contrast-ratio

Closes #23454




## What I did

The contrast ratio of the RAW button was not >= 4.5, so I changed the color of the button to achieve this. I have used a darker color from the existing colors which are used in the application. 

## Checklist for Contributors

### Testing


#### The changes in this PR are covered in the following automated tests:

#### Manual testing

1. Run the application using yarn start
2. Open Storybook in your browser
3. Navigate to the app bar present in the left pane using tab key and invoke it using Enter key.
4. In Search Components search-bar, search for 'inline', and select Padded Inline.
5. Now navigate to “Controls” tab. There you can see RAW button. Check the contrast ratio of this button. It should be greater than equal to 4.5. With my change, the contrast ratio is now 5.72


### Documentation

      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
